example.py

```diff
@@ -14,52 +13,56 @@ def __len__(self):
         return 1
 
     def __getitem__(self, idx):
-        image_path = '/Users/michael/repo/VideoGAN/Ms_Pacman/Test/0000/00070.png'
+        image_path = "/Users/michael/repo/VideoGAN/Ms_Pacman/Test/0000/00070.png"
```

Comment:
Can we not do this absolute filepaths?
Or are will we remove this file altogether? Is it needed?